delta-api/src/main/java/io/cdap/delta/api/DeltaRuntimeContext.java

```diff
@@ -79,4 +80,9 @@
    * @return the id of the delta pipeline
    */
   DeltaPipelineId getPipelineId();
+
+  /**
+   * @return all the Tables to be replicated
```

Comment:
nit: all the tables to be replicated by the current pipeline

Reply:
done:)